build_scripts/build_dependencies_windows_cuda.ps1

```diff
@@ -33,4 +55,9 @@ if (-not (Test-Path "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\$cudaVer
     Copy-Item -Path "C:\Users\$userName\Downloads\cudnn-*\bin\*.dll" -Destination "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\$cudaVersion\bin\" -Recurse
 }
 
-Write-Host "Dependencies verified and installed successfully."
\ No newline at end of file
+Write-Host "Dependencies verified and installed successfully."
+
+# Build Documentation
```

Comment:
We Should not build documentation while building dependencies

Reply:
Removed documentation build from scripts.